commands/fun/anime.js

```diff
@@ -24,18 +24,18 @@ class Anime extends Command {
       return ["1", "2", "3", "4", "5"].includes(msg.content);
     }
     if (args.length < 1) return message.reply("You must add an anime to search for");
-    let msg = await message.channel.send("*fetching information from kitsu!*");
+    let msg = await message.channel.send("Fetching information from Kitsu!");
     try {
       const { data } = await kitsu.fetch("anime", { filter: { text: args.join("-") } });
       if (data.length < 1) throw new this.client.methods.errors.AnimeError("No result found");
-      msg = await msg.edit(`Okay i found 5 possible matches which do you want to see? (just write the first number, it will be canceled after 60 seconds)${this.makeTitles(data)}`);
+      msg = await msg.edit(`Okay, I found 5 possible matches!~ Which one do you want to see~\n(Just write the number, the one you want to see!)${this.makeTitles(data)}`);
       const collected = await message.channel.awaitMessages(filter, { max: 20, maxProcessed: 1, time: 60000, errors: ["time"] });
       const returnMessage = collected.first();
       const index = Number(returnMessage.content) - 1;
       if (message.channel.permissionsFor(this.client.user).has("MANAGE_MESSAGES")) await returnMessage.delete(); 
       await msg.edit(`**Title JP:** ${data[index].titles.en_jp}\n**Title English:** ${data[index].titles.en}\n**Type:** ${data[index].subtype}\n**Start Date:** ${data[index].startDate}\n**End Date:** ${data[index].endDate || "in Progress"}\n**PopularityRank:** ${data[index].popularityRank}\n**Link:** <https://kitsu.io/anime/${data[index].id}>\n**Synopsis:** ${data[index].synopsis}`);
     } catch (error) {
-      if (error instanceof Collection) return message.reply("command canceled due timer");
+      if (error instanceof Collection) return message.reply("Command canceled due to the timer~");
```

Comment:
Due to the nature of `*.reply()`, I do not recommend changing "command" to "Command".